ci-operator/step-registry/cucushift/installer/cucushift-installer-ref.metadata.json

```diff
@@ -0,0 +1,13 @@
+{
+	"path": "cucushift/installer/cucushift-installer-ref.yaml",
```

Comment:
Maybe we can place a new folder (named "check" ?) under `cucushift/installer/` to save the new ref?

Reply:
Yes, that's better idea.